package/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN make -C /helm
 FROM alpine:3.12 AS build
 ARG ARCH=amd64
 RUN apk add -U curl xz
-ENV KUBECTL_VERSION v1.19.7
+ENV KUBECTL_VERSION v1.21.8
```

Comment:
Did you confirm that this is valid with k8s 1.17 ?

Reply:
Yes, checked using Rancher Desktop with Kubernetes v1.17.17 and kubectl v1.21.8.
On regular operations, like `kubectl run/get/describe`, there are no errors or warnings.
There is a warning, however, in `kubectl version`:

```
./kubectl version --short               
Client Version: v1.21.8
Server Version: v1.17.17+k3s1
WARNING: version difference between client (1.21) and server (1.17) exceeds the supported minor version skew of +/-1
```

Reply:
Oh so officially its +/- 1, then doing 1 down from the latest makes most sense, which for 2.6.4 is 1.21.